modules/metastore/src/Exception/AlreadyRegistered.php

```diff
@@ -12,7 +12,7 @@ class AlreadyRegistered extends \Exception {
   /**
    * An array of entities that have already been registered in the mapping.
    *
-   * @var \Drupal\Core\Entity\EntityInterface[]
+   * @var \Drupal\Core\Entity\ContentEntityInterface[]
```

Comment:
We call `get()` from `$alreadyRegistered` over in `Drupal\metastore\Reference\Referencer` so general `EntityInterface` is not specific enough/